### PR TITLE
Fix elevenlabs voice settings breaking

### PR DESCRIPTION
**Pull request by nbsp**
11labs API requires `style` float and `use_speaker_boost` boolean. defaulting to None gets translated to JSON as `null` type. this sets them to their default values noted in the API here: https://api.elevenlabs.io/v1/voices/settings/default

### CODE REVIEW

**Comment by nbsp** [comment]
just noticed that this does break our API, as the default for speaker boosting is now `True` and it used to be `False`; do we keep it as it was?

**Comment by theomonnom** [comment]
Are the defaults safe when listing the voices using this fnc:
  async def list_voices(self) -> List[Voice]:

**Comment by nbsp** [comment]
> Are the defaults safe when listing the voices using this fnc?
> `async def list_voices(self) -> List[Voice]:`

that function sets `settings=None`, which changes the behaviour of the stock voices now, yes

**Comment by theomonnom** [comment]
Looking at their docs, they seem to explicitly use it on the examples? https://github.com/elevenlabs/elevenlabs-python/blob/a75c7e7e0b2aa8d04225ca34e2e387ccd0ba13ac/README.md#%EF%B8%8F-voices

On their 11labs python-client, I see that they also default these stuff to None

File: livekit-plugins/livekit-plugins-elevenlabs/livekit/plugins/elevenlabs/tts.py [hunk 1]
```diff
@@ -50,8 +50,8 @@ def _encoding_from_format(output_format: TTSEncoding) -> _Encoding:
 class VoiceSettings:
     stability: float  # [0.0 - 1.0]
     similarity_boost: float  # [0.0 - 1.0]
-    style: float | None = None  # [0.0 - 1.0]
-    use_speaker_boost: bool | None = False
+    style: float = 0.0  # [0.0 - 1.0]
+    use_speaker_boost: bool = True
```
Comment (keepingitneil) [inline livekit-plugins/livekit-plugins-elevenlabs/livekit/plugins/elevenlabs/tts.py:54]: should we keep the same default as before?

**Review by theomonnom — APPROVED** [review]

**Comment by nbsp** [comment]
new method just strips the `None`s from the object before passing it to the JSON encoder, and doesn't mess with our defaults, so everything should be the same.


> Looking at their docs, they seem to explicitly set it on the examples? [elevenlabs/elevenlabs-python@`a75c7e7`/README.md#%EF%B8%8F-voices](https://github.com/elevenlabs/elevenlabs-python/blob/a75c7e7e0b2aa8d04225ca34e2e387ccd0ba13ac/README.md#%EF%B8%8F-voices)
> 
> On their 11labs python-client, I see that they also default these stuff to None

i don't think elevenlabs' Python client even supports professional or cloned voices, they errored as well.

**Pull request merged**